File: testdata/expected/valswithtypes/FilledArray.scala
class FilledArray
{
  var a : Array[Int] = Array[int]( 1, 2, 3 )
  
  var b : Array[Object] = Array[Object]( 
    "1", 
    "2", 
    "3" 
  )

  var inner : Array[Object] = Array[Object]( "1", "2",
    Array[Object]( "a", "b" ) )  
}

File: testdata/FilledArray.java
class FilledArray
{
  int[] a = new int[] { 1, 2, 3 };
  
  Object[] b = new Object[] { 
    "1", 
    "2", 
    "3" 
  };

  Object[] inner = new Object[] { "1", "2",
    new Object[] { "a", "b" } };  
}